test/Microsoft.AspNetCore.Razor.Runtime.Test/Runtime/TagHelpers/CaseSensitiveTagHelperAttributeComparer.cs

```diff
@@ -25,8 +25,8 @@ public bool Equals(TagHelperAttribute attributeX, TagHelperAttribute attributeY)
             // Normal comparer (TagHelperAttribute.Equals()) doesn't care about the Name case, in tests we do.
```

Comment:
Old code but... shouldn't this class also "care" about the `Name` case in `GetHashCode()`?